Library/Formula/openshift-cli.rb

```diff
@@ -18,16 +18,28 @@ class OpenshiftCli < Formula
 
   def install
     # this is necessary to avoid having the version marked as dirty
-    (buildpath/".git/info/exclude").atomic_write "/.brew_home"
+    (buildpath/".git/info/exclude").atomic_write <<-EOF
+/.brew_home
+/src
+    EOF
 
-    system "make", "all", "WHAT=cmd/openshift", "GOFLAGS=-v"
+    # this is a terrible hack that's necessary because the build script assumes
+    # that the source code was checked out via `go get` and overrides $GOPATH,
+    # and also because make will change into the target of folder symlinks
+    # rather than the folder symlinks themselves
+    real_buildpath = buildpath/"src/github.com/openshift/origin"
+    mkdir_p real_buildpath
```

Comment:
You don’t need this line, `real_buildpath` will be created by `real_buildpath.install` below.